Gw2 Launchbuddy/MainWindow.xaml.cs

```diff
@@ -1914,12 +1914,12 @@ private void UpdateServerArgs()
 
         private void Window_LostKeyboardFocus(Object sender, System.Windows.Input.KeyboardFocusChangedEventArgs e)
         {
-            if (cinemamode || rb_cinemavideomode.IsChecked == true) { Cinema_MediaPlayer.Pause(); }
+            if (cinemamode | rb_cinemavideomode.IsChecked == true) { Cinema_MediaPlayer.Pause(); }
             BeginStoryboard(this.FindResource("anim_musicfadeout") as System.Windows.Media.Animation.Storyboard);
         }
         private void Window_GotKeyboardFocus(Object sender, System.Windows.Input.KeyboardFocusChangedEventArgs e)
         {
-            if (cinemamode || rb_cinemavideomode.IsChecked == true) { Cinema_MediaPlayer.Play(); } 
```

Comment:
I'm confused on this change. Why would you want to evaluate the second operand if the first is true?